types/node/assert.d.ts

```diff
@@ -11,11 +12,37 @@ declare module "assert" {
             code: 'ERR_ASSERTION';
 
             constructor(options?: {
-                message?: string; actual?: any; expected?: any;
-                operator?: string; stackStartFn?: Function
+                /** If provided, the error message is set to this value. */
+                message?: string;
+                /** The `actual` property on the error instance. */
+                actual?: any;
+                /** The `expected` property on the error instance. */
+                expected?: any;
+                /** The `operator` property on the error instance. */
+                operator?: string;
+                /** If provided, the generated stack trace omits frames before this function. */
+                stackStartFn?: Function
             });
         }
 
+        class CallTracker {
+            calls(exact?: number): () => void;
+            calls<Func = (...args: any[]) => any>(fn?: Func, exact?: number): Func;
+            report(): CallTrackerReportInformation[];
+            verify(): void;
+        }
+        export interface CallTrackerReportInformation {
+            message: string;
+            /** The actual number of times the function was called. */
+            actual: number;
+            /** The number of times the function was expected to be called. */
+            expected: number;
+            /** The name of the function that is wrapped. */
+            operator: string;
+            /** A stack trace of the function. */
+            stack: object;
```

Comment:
I think this is actually an `Error` object

Reply:
[Looks like that.](https://github.com/nodejs/node/blob/636c0bb419907188308c01992e546ab1628bc556/lib/internal/assert/calltracker.js#L40)

Reply:
[Looks like that.](https://github.com/nodejs/node/blob/636c0bb419907188308c01992e546ab1628bc556/lib/internal/assert/calltracker.js#L40)